Tests/CreateAPITests/Specs/petstore.yaml

```diff
@@ -120,3 +120,14 @@ components:
           format: int32
         message:
           type: string
+    Dog:
+      allOf:
+        - $ref: "#/components/schemas/Pet"
+        - type: object
+          properties:
+            breed:
+              type: string
+              enum:
+              - "Large"
+              - "Medium"
+              - "Small"
```

Comment:
I think that it's fine to remove this again now that we've verified that it works in that scenario that I described. Otherwise you'll have to regenerate all of the other fixtures that rely on **petstore.yaml** too.

Reply:
Sorry, I didn't see this before I committed, oops. If you'd like them removed I can revert.